### ISSUE TITLE
Suggestions to polish to AR part of Nomi

### ISSUE THREAD

**Issue by ChromaPIE** [Nomi-CEu/Nomi-CEu#1124]
### Nomi CEu Version

Nightly

### Launcher

(not related)

### Environment

(not related)

### Mode

Both Modes

### Related Problem

Many AR stuff got left behind unmodified, unpolished. I respect the idea that Nomi is not about exploring and AR is just an optional part of the pack but still, you either completely wipe them out of the game or make it better, there's no point leaving if there.

### Your Solution

- Geode blocks generated on Moon and some other planets can only be harvested with Jackhammer, which has been disabled/hidden. It's ugly and useless so if you want to get inside you can just hold LMB for a minute and destroy one, but it will cause confusion, because TOP will say Jackhammer (Diamond) when you point at it.
- Also the geode sphere. There are vanilla and Thermal ores generated inside and they show on the Prospector. I believe it can be tweaked to at least generate GT ores.
- You **can** reach other planets, but there's nothing there other than different types of terrain, sparse Draconic ores, and Geode balls mentioned above. Rice is even being generated on Venus. It's giving *No one, not even the Dev plays AR on Nomi so just go back to your base*. So I'm thinking about, Aluminum ore is obtainable post-Ender, how about making some of the planets generate natural Titanium ores or some other fun stuff to show respect to players that managed to get onto the planet? Because, well, the intergalactic thingies are not removed or marked as Deprecated, the Warp Core is even on the quest book.
- And maybe more.

### How Will Your Solution Work

.

**Comment by D-Alessian** [comment]
I agree that AR really isn't polished, but I don't agree that it is for those reasons. I never came across geodes myself, so I can't speak on that but doesn't seem like that much of a problem. As for other planets, they're not supposed to be used at all, only the moon is supposed to, space exploration isn't meant to be a part of Nomi. If you can disable planets, maybe that would be a good option, but adding content to other planets doesn't really fit imo. 

What is more annoying imo is that the rocket building process feels clunky, the items look old and just don't feel good to use, the multiblocks you can make like the warp drive for space station also feel weird to use. The moon gets visual artefacts sometimes too where the floor is transparent at some spots, and not on others. It overall doesn't feel like it's a well implemented part of Nomi and more of a temp fix.

**Comment by ChromaPIE** [comment]
Removing/Banning planets is indeed an once and for all solution. I'm totally not against the idea of simply doing so. My point is that if you're allowing players to give them a visit, which, might not be intended but just because that no one is really giving an F to AR in Nomi, is what the pack is doing at current state, then you should polish it, like stop them from generating Draconic Ores, or just wipe them out.

For the Geode part, it might be because that I'm using the forked version of AR which is said to have some bugs fixed and it might be unexpectedly generating the geodes, if so just ignore those parts mentioning it, but besides this everything works just fine.
